api/src/flows.ts

```diff
@@ -233,6 +233,7 @@ class FlowManager {
 
 			return { successor: operation.resolve, data: result ?? null };
 		} catch (error: unknown) {
+			if (!operation.reject) logger.warn(`Unhandled error in operation "${operation.key}": ${error}`);
```

Comment:
Not sure if we should always warn in case the rejection of an operation isn't handled. There might be cases where it is perfectly fine to simply stop the execution of a flow, e.g. when using the `validate` operation as an "if" without an "else".

Reply:
Good point!